docs/features/logforwarding/outputs/splunk-forwarding.md

```diff
@@ -0,0 +1,66 @@
+
+### Steps to forward to Splunk HTTP Event Collector (HEC)
+(https://dev.splunk.com/enterprise/docs/devtools/httpeventcollector/)
+
+1. Create a secret containing your base64-encoded `hecToken` using this example command.
+    ```
+    sed "s/<TOKEN>/$(echo -n hec_token_here | base64 -w0)/g" splunk-secret.yaml | oc apply -f -
+    ```
+    Where `splunk-secret.yaml` contains the following:
```

Comment:
we have this command 'oc create secret generic'. Why not prefer that over a template ?

Reply:
Sure, done